_sass/custom/custom.scss

```diff
@@ -187,3 +187,68 @@ span.na {
 .search-results {
   width: 95%;
 }
+
+.alert {
```

Comment:
I'm no fan of having styles which are not in use.

I can't see any `h4`, `.alert-link`, `ul` or `hr` elements. And - if we go with additional style elements - they should match the existing styles. In this case, I'd remove the border to match the codeblocks.

Reply:
removed